Library/Formula/tippecanoe.rb

```diff
@@ -23,8 +23,10 @@ def install
     path.write <<-EOS.undent
       {"type":"Feature","properties":{},"geometry":{"type":"Point","coordinates":[0,0]}}
     EOS
-    output = `#{bin}/tippecanoe -o test.mbtiles #{path}`.strip
+
+    `#{bin}/tippecanoe -o test.mbtiles #{path}`
```

Comment:
Aye, Mike's comment is what I was hinting at. You can try `shell_output` if `system` is failing as well.